src/main/java/com/adyen/model/nexo/SaleData.java

```diff
@@ -75,7 +79,8 @@
      * The Sale to acquirer data.
      */
     @XmlElement(name = "SaleToAcquirerData")
-    protected String saleToAcquirerData;
```

Comment:
Removing/changing type saleToAcquirerData field means release won't be backward compatible.

Reply:
correct I don't see any other way.